### PR TITLE
Pair programming(cooperative job)

### PR DESCRIPTION
**Pull request by er-contreras**
- Today we work on our respective projects helping us and commit together in every one project.
- We help to build part of the mobile version using forms!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,210 +1,272 @@
...
-    </div>
+        <fieldset>
+          <label for="name"></label>
+          <input type="text" name="name" id="name" placeholder="Text input" minlength="5" maxlength="30" required/>
+  
+          <label for="email"></label>
+          <input type="email" name="email" id="email" placeholder="Email address" required/>
 
-    <button type="button">See Project</button>
+          <textarea name="user-message" placeholder="Enter text here" minlength="3" maxlength="500" required></textarea>
```
Comment (Davidosky007) [inline index.html:237]: I suggest message input's type equals textarea as stated in the project requirements

File: index.html [hunk 2]
```diff
@@ -1,210 +1,272 @@
...
+        </fieldset>
 
-  </main>
+        <input class="btn-center" type="submit" value="See Project">
```
Comment (Davidosky007) [inline index.html:240]: I suggest you make this button have the necessary interactions such as hover, active, disable and enable effects as shown in the Figma template

File: index.html [hunk 3]
```diff
@@ -1,210 +1,272 @@
...
+        </div>
+
+        <div>
+          <img src="/icons/M.png" alt="medium" />
+        </div>
+      </div>
+
+      <div class="link-email">
+        <a href="#">eri89--@hotmail.com</a>
+      </div>
```
Comment (Davidosky007) [inline index.html]: I suggest you wrap these social link list in a `ul` tag and each social link with `li` tag this is best practice

File: index.html [hunk 4]
```diff
@@ -1,210 +1,272 @@
...
+        <div>
+          <img src="/icons/2.png" alt="Angelist" />
+        </div>
+
+        <div>
+          <img src="/icons/twitter.png" alt="twitter" />
+        </div>
+
+        <div>
+          <img src="/icons/M.png" alt="medium" />
```
Comment (Davidosky007) [inline index.html]: * Each social link icon should be an `a` tag as this is best practice 
* your social links are broken please fix them
Screenshot (27)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @eri8-9 @Richie9000, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I left you comments below please make the necessary changes and submit for review 👍 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (er-contreras) [reply index.html:237]: @Davidosky007 Sry can you be more explicit, I don't understand. do you mean give textarea a label an put the same name?

Comment (Davidosky007) [reply index.html:237]: ` <textarea name="user-message" type="textarea" placeholder="Enter text here" minlength="3" maxlength="500" required></textarea>`   (something like this )

Comment (er-contreras) [reply index.html:237]: @Davidosky007 I run npx hint . but give a error with that changes.
do you have other suggestion?
  
Screen Shot 2021-06-08 at 20 32 11

Comment (Davidosky007) [reply index.html:237]: ok you can remove it 🙏 continue working on the other changes 👍

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**